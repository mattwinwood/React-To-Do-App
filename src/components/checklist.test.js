import React from 'react';
import renderer from 'react-test-renderer';
import Checklist from './checklist';
import tasksMocks from '../__mocks__/tasks';

it('renders correctly', () => {
        const tree = renderer.create(<Checklist tasks={tasksMocks}/>).toJSON();
        expect(tree).toMatchSnapshot();
    });
