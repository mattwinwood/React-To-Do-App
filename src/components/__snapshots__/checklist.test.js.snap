// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    className="alert alert-success undefined"
    role="alert"
  >
    Completed 
  </div>
  <ul
    className="list-group"
  >
    <li
      className="list-group-item"
      onClick={[Function]}
      onMouseUp={[Function]}
      value="not_started"
    >
      <span
        className="circle active"
      />
      <span
        className="todo-desc"
      >
        To do task #1
      </span>
    </li>
    <li
      className="list-group-item"
      onClick={[Function]}
      onMouseUp={[Function]}
      value="not_started"
    >
      <span
        className="circle active"
      />
      <span
        className="todo-desc"
      >
        To do task #2
      </span>
    </li>
    <li
      className="list-group-item"
      onClick={[Function]}
      onMouseUp={[Function]}
      value="not_started"
    >
      <span
        className="circle active"
      />
      <span
        className="todo-desc"
      >
        To buy task #1
      </span>
    </li>
  </ul>
  <h2
    className="sc-bdVaJa geYwAp"
  >
    2
     tasks completed
  </h2>
  <ul
    className="list-group"
  >
    <li
      className="list-group-item"
      onClick={[Function]}
      onMouseUp={[Function]}
      value="complete"
    >
      <span
        className="circle complete"
      />
      <span
        className="todo-desc"
      >
        To do task #3
      </span>
    </li>
    <li
      className="list-group-item"
      onClick={[Function]}
      onMouseUp={[Function]}
      value="complete"
    >
      <span
        className="circle complete"
      />
      <span
        className="todo-desc"
      >
        To do task #4
      </span>
    </li>
  </ul>
</div>
`;
